examples/terraform/digitalocean/variables.tf

```diff
@@ -38,6 +38,11 @@ variable "ssh_port" {
   description = "SSH port to be used to provision instances"
 }
 
+variable "ssh_user" {
+  default = "root"
+  description = "SSH user, used only in output"
```

Comment:
It isn't only used in the output, but it's also used by KubeOne to connect to the instances. Maybe we should improve the description to mention that.

Reply:
yes, that's what outputs are used